### ISSUE TITLE
Don't auto scroll to start

### ISSUE THREAD

**Issue by nico202** [voldyman/MarkMyWords#37]
When I edit the text, i find the preview automatically going up to the start of the document, so that I cannot see the preview of what I am writing.

**Comment by emersion** [comment]
Didn't find the method to get/set scroll value with Webkit2gtk.

**Comment by nico202** [comment]
It's really annoying, hope there will be a way to do this. Also, the same happen on the left part of the screen (the editable one) when I save a long file; I think at least this should be easy to solve.

**Comment by emersion** [comment]
I agree. I tried to find a `scroll` property one more time, but I don't find one in [the docs](http://valadoc.org/#!api=webkit2gtk-4.0/WebKit.WebView). It turns out that we'll have to do this with some javascript...
